eZ/Publish/API/Repository/Repository.php

```diff
@@ -41,7 +43,7 @@ public function sudo(callable $callback, Repository $outerRepository = null);
      *
      * @return \eZ\Publish\API\Repository\ContentService
```

Comment:
This could be removed as we have a return type hint here and below

Reply:
I will remove it as follow up. Its general issue in PAPI.